src/pages/ProductList/ProductList.js

```diff
@@ -1,9 +1,132 @@
 import React, { Component } from 'react';
+import Product from './productListComponent/Product';
+import Footer from '../../components/Footer/Footer';
 import './ProductList.scss';
+import '../ProductList/productListComponent/product.scss';
 
 class ProductList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      productData: [],
+      productStyle: '',
+    };
+  }
+
+  componentDidMount() {
+    fetch('./data/productListData-doyoung.json', {
+      method: 'GET',
+    })
+      .then(res => {
+        return res.json();
+      })
+      .then(res => {
+        this.setState({
+          productData: res,
+        });
+      });
```

Comment:
```suggestion
      .then(res => res.json())
      .then(res => this.setState({ productData: res }));
```
간결하게 작성할 수 있는 부분이라면 가급적 그렇게 하는게 좋겠습니다